tests/test_ci/TestControllers.py

```diff
@@ -14,7 +15,138 @@
                         generate_signature, mock_api_request_github)
 
 
+class MockKVM:
+
+    def __init__(self, name):
+        self.name = name
+
+
+class MockPlatform:
+
```

Comment:
Wouldn't pylint return a error if the Class doesn't have documentation?

Reply:
Test classes are excluded if I remember correctly :)